metadata.json

```diff
@@ -5,12 +5,14 @@
   "title": {
     "en_EN": "Preview",
     "nb_NO": "Forhåndsvisning",
-    "fr_FR": "Outil de prévisualisation"
+    "fr_FR": "Outil de prévisualisation",
+    "pt_BR": "Preview"
```

Comment:
And this one not translatable either ? :blush:

Reply:
That word yes, but we use "Preview" frequently in Brazil, just like "Music Player"

Reply:
Okay. Then you can go by the comment I left in the music player app repo :)

Reply:
So you can merge this too :)